jenkinsfiles/dev

```diff
@@ -194,6 +194,7 @@ pipeline {
                 IMAGE_PULL_POLICY = "Always"
                 FLYWAY_MIGRATE_OUT_OF_ORDER = "true"
                 FLYWAY_REPAIR_BEFORE_MIGRATION = "true"
+                INSTANCE_TTL = "315360000"
```

Comment:
10 years seems a bit excessive 😄, but it shouldn't really matter.

Reply:
Time flies :smile: